src/transformers/models/llava_next/processing_llava_next.py

```diff
@@ -166,6 +166,8 @@ def __call__(
                         if not isinstance(image_size, (list, tuple)):
                             # cast to list to avoid numerical precision errors when calculating unpadding
                             orig_height, orig_width = image_size.tolist()
+                        else:
+                            orig_height, orig_width = image_size
```

Comment:
ultra nit: could be more readable as this 

```python
if not isinstance(image_size, (list, tuple)):
    # cast to list to avoid numerical precision errors when calculating unpadding
    image_size = image_size.tolist()
orig_height, orig_width = image_size
```